authorization.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import Header, HTTPException
+
+# Define available roles
+VALID_ROLES = {"admin", "user"}
```

Comment:
create a separate file called constants.py, 
add all the hard codings there and import them here